modules/virt-editing-vm-web.adoc

```diff
@@ -28,18 +28,19 @@ endif::[]
 = Editing a {object} in the web console
 
 Edit select values of a {object} in the *{object-gui} Overview* screen
-of the web console by clicking on the pencil icon next to the relevant field. 
-Other values can be edited using the CLI. 
+of the web console by clicking on the pencil icon next to the relevant field.
+Other values can be edited using the CLI.
 
 .Procedure
 
-. Click *Workloads* -> *{object-gui}s* from the side menu.
-. Select a {object} to open the *{object-gui} Overview* screen.
-. Click the pencil icon to make that field editable.
+. Click *Virtualization* from the side menu.
+. Select the {object} tab to open the *{object-gui} Overview* screen.
+. Select the *{object}* you want to edit.
+. Click the pencil icon to make a field editable.
 . Make the relevant changes and click *Save*.
```

Comment:
VM overview tab serves as the VM dashboard, no fields are editable on overview. `Details` tab has editable fields, it requires the VM is stopped on OCP 4.5.
Suggest steps:
1. Click *Virtualization* from the side menu.
2. Select the *Virtual Machine* tab.
3. Click the virtual machine name to open the *Virtual Machine Overview*.
4. Click “Details" tab.
5. Select the *{object}* you want to edit.
6. Click the pencil icon to make a field editable.
7. Make the relevant changes and click *Save*.